test/cpp/end2end/async_end2end_test.cc

```diff
@@ -2013,6 +2012,7 @@ INSTANTIATE_TEST_CASE_P(AsyncEnd2endServerTryCancel,
 }  // namespace grpc
 
 int main(int argc, char** argv) {
+  gpr_setenv("GRPC_CLIENT_CHANNEL_BACKUP_POLL_INTERVAL_MS", "200");
```

Comment:
Please add a comment for context as why this is needed.

Reply:
Added. Thanks for the review!